packages/tpetra/core/src/Tpetra_Details_PackTraits.hpp

```diff
@@ -249,12 +275,18 @@ struct PackTraits {
       // entries of the input array have the correct size.
       const T& val = packValueCount (outBuf(0));
```

Comment:
I don't understand how `const T& val = packValueCount(outBuf(0))` ever worked.  The function returns `size_t`, not `T`.